src/general/OptionsMenu.tscn

```diff
@@ -297,6 +298,7 @@ size_flags_vertical = 0
 selected = 0
 item_count = 3
 popup/item_0/text = "WINDOWED"
+popup/item_0/id = 0
```

Comment:
Could you try reopening Godot and resaving this as I think that might remove these default value assignments?

Reply:
Right, this is new or at least with Godot 4.4 now I also get this. So please keep these changes to make the settings scene be fixed up for 4.4.